covid19-etl/utils/metadata_helper.py

```diff
@@ -106,7 +106,7 @@ def get_str_latest_submitted_date_idph(self):
             )
         loc = query_res["data"]["location"]
         if (len(loc)) > 0:
-            return loc[0]["date"]
+            return loc[0]["date"].split("T")[0]
```

Comment:
It was my mistake. I guess! Previously all the `date` fields were in `YYYY-MM-DD` format. While I was running this etl in QA, I pushed a `YYYY-MM-DD T HH:MM:SS` format. I needed to add this for that correction. Later I wanted to keep this because it was better to be safe than sorry! :/ 

Thoughts?

Reply:
👍